.~lock.test1.xls#

```diff
@@ -0,0 +1 @@
+,jbarry465,ubuntu,23.02.2015 14:48,file:///home/jbarry465/.config/libreoffice/4;
```

Comment:
I'll merge this and delete this file when I'm home, but make sure next time you only 'git add' files that need to be in the repo :smile_cat: